examples/exotica_examples/resources/configs/ik_solver_demo.xml

```diff
@@ -26,6 +26,8 @@
         </EndEffector>
       </EffFrame>
     </Maps>
+    <StartState>0 0 0 0 0 0 0</StartState>
+    <NominalState>0 0 0 0 0 0 0</NominalState>
```

Comment:
We may want to use the usual nominal pose here with the bent arm?

Reply:
I prefer zeros for simplicity. Any other value would require a comment explaining what pose that is.

Reply:
Shouldn't nominal default to zeros or middle-of-joint-range if not set?

Reply:
No, no nominal pose vector will result in not using the null space at all (min-norm).